### PR TITLE
Modify BaseShapes for correct 3D visualization of particles in GUI

### PR DESCRIPTION
**Pull request by anikhalder**
Note: Dot, Ripple1 and Ripple2 shapes not yet implemented.

### CODE REVIEW

File: GUI/ba3d/ba3d/model/particles.cpp [hunk 1]
```diff
@@ -54,133 +54,126 @@ void Particle::fancy(Vector3D rotate, float r)
...
+static float const pi   = float(M_PI);
 static float const pi2f   = float(M_PI_2);
 static float const sqrt2f = std::sqrt(2.f);
 static float const sqrt3f = std::sqrt(3.f);
 
-FullSphere::FullSphere(float R)
-    : Particle(Key(BaseShape::Sphere, 0))
+// see ~/BornAgain/GUI/ba3d/ba3d/model/geometry/ for BaseShape construction
+
+AnisoPyramid::AnisoPyramid(float L, float W, float H, float alpha)
```
Comment (gpospelov) [inline GUI/ba3d/ba3d/model/particles.cpp]: Next time please do such things in two separate commits. First reorder alphabetically, then change parameters. Otherwise hard to track changes...

Comment (anikhalder) [reply GUI/ba3d/ba3d/model/particles.cpp]: Thank you. I shall take note of that.
